xml/System.IO/FileAttributes.xml

```diff
@@ -47,18 +47,31 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- You can get attributes for files and directories by calling the <xref:System.IO.File.GetAttributes%2A> method, and you can set them by calling the <xref:System.IO.File.SetAttributes%2A> method.  
+ You can get attributes for files and directories by calling the <xref:System.IO.File.GetAttributes%2A?displayProperty=nameWithType> method, and you can set them by calling the <xref:System.IO.File.SetAttributes%2A?displayProperty=nameWithType> method.  
   
- It is not possible to change the compression status of a <xref:System.IO.File> object by using the <xref:System.IO.File.SetAttributes%2A> method. Instead, you must actually compress the file using either a compression tool or one of the classes in the <xref:System.IO.Compression?displayProperty=nameWithType> namespace.  
-  
-   
+ It is not possible to change the compression status of a <xref:System.IO.File> object by using the <xref:System.IO.File.SetAttributes%2A?displayProperty=nameWithType> method. Instead, you must actually compress the file using either a compression tool or one of the classes in the <xref:System.IO.Compression?displayProperty=nameWithType> namespace.  
   
+The following attributes are not supported by .NET Core on Linux and macOS:
+
+- <xref:System.IO.FileAttributes.Archive?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.Compressed?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.Device?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.Encrypted?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.IntegrityStream?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.NoScrubData?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.NotContentIndexed?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.Offline?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.SparseFile?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.System?displayProperty=nameWithType>
+- <xref:System.IO.FileAttributes.Temporary?displayProperty=nameWithType>
+
+On Unix systems, the value returned by <xref:System.IO.File.GetAttributes%2A?displayProperty=nameWithType> includes `Hidden` for a file whose name begins with a period ("."). On macOS, you can get or set the hidden flag.
+
 ## Examples  
  The following example shows how to retrieve the attributes for a file and check if the file is read-only.  
   
  [!code-csharp[System.IO.FileAttributes#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.io.fileattributes/cs/example.cs#1)]
- [!code-vb[System.IO.FileAttributes#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.io.fileattributes/vb/example.vb#1)]  
-  
+ [!code-vb[System.IO.FileAttributes#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.io.fileattributes/vb/example.vb#1)]   
```

Comment:
I think you need that new line between the snippet and the end of the cdata section for loc.

```suggestion
 [!code-vb[System.IO.FileAttributes#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.io.fileattributes/vb/example.vb#1)] 

```